presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -291,8 +286,23 @@ void PrestoServer::run() {
   registerVectorSerdes();
   registerPrestoPlanNodeSerDe();
 
+  ioExecutor_ = std::make_shared<folly::IOThreadPoolExecutor>(
```

Comment:
How about we call it exchangeExecutor_ as it is used for async exchange operations over http? Thanks!